Automated QA tests (#101)
taiwangoldcard/website: README.md

- Output: `docs/` is the "build" folder, where GitHub look at to show the [Taiwan Gold Card](https://taiwangoldcard.com/) website

> ### Running automated tests
> 
> All tests are ran automatically via [github workflows](./.github/workflows/tests.yml) when you push a commit. However, you can still run the tests on local by following these steps:

Reviewer: ```suggestion
### Tests
We have tests to check how if the website still works before beeing available to users. All tests are ran automatically via [github workflows](./.github/workflows/tests.yml) when you push a commit. You can see them [here](https://dashboard.cypress.io/projects/rtyzr7/runs), each test has a video attached to it. 

If you need to run the tests locally:
```

1. First, install node / npm. This is required for the test executable. You may use [Node Version Manager](https://github.com/nvm-sh/nvm):

```
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```

2. At this repo's root, perform `npm install`. This will install `cypress`, the automated test runner.

> 3. `cypress run` to perform all tests in headless mode / CI mode. 

Reviewer: @geekyme  maybe this one would work better since cypress isn't installed globally: `./node_modules/cypress/bin/cypress run`? 

an idea for an other PR:   `hugo server` needs to be started to run the tests locally. We could perhaps add a `npm test` command that do both command together?

Author: weird i was able to do `cypress run`. Does `npx cypress run` work for you?

> 3. `cypress run` to perform all tests in headless mode / CI mode. 

Reviewer: ```suggestion
3. `./node_modules/cypress/bin/cypress run` to perform all tests in headless mode / CI mode. 
```

> 4. To debug tests, you can `cypress open` to view the interactive UI and select the tests to run:

Reviewer: ```suggestion
4. To debug tests, you can `./node_modules/cypress/bin/cypress open` to view the interactive UI and select the tests to run:
```
